### PR TITLE
MNT: add maintainer

### PR DESCRIPTION
**Pull request by kmuehlbauer**
ping @groutr: shall I add you here as maintainer?

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,4 +29,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - kmuehlbauer
\ No newline at end of file
+    - kmuehlbauer
+    - ocefpaf
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Let's wait for @groutr to see if he wants in too.

**Comment by ocefpaf** [comment]
Thanks @kmuehlbauer!

File: recipe/meta.yaml (hunk 1 above)
Comment (kmuehlbauer) [inline recipe/meta.yaml]: Sure!

**Comment by ocefpaf** [comment]
Ping @pelson: I love the fact that AppVeyor did not start here :wink:

**Comment by kmuehlbauer** [comment]
ping @groutr: Do you use netcdf-c++ bindings at all? It would be good, to have you with the maintainers here.

**Pull request merged**